packages/lib-classifier/src/plugins/drawingTools/experimental/models/marks/FreehandLine/FreehandLine.js

```diff
@@ -67,8 +67,13 @@ const FreehandLineModel = types
         path.push(pointPath)
       })
       // closes the drawing path
-      if (!self.dragPoint && self.isClosed) {
-        path.push(`Z`)
+      if (self.isClosed) {
+        if (self.dragPoint) {
+          const { x, y } = self.initialPoint
+          path.push(`L ${x},${y}`)
+        } else {
+          path.push(`Z`)
+        }
```

Comment:
The [`Z` path command](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d#closepath) closes the current subpath, which looks very weird when the line is cut into two subpaths. This works around that by drawing a line to the first point when the line has been cut.